styles.css

```diff
@@ -99,4 +99,269 @@ nav {
   font-size: 24px;
   padding: 10px;
   align-items: stretch;
+  padding-bottom: 20px;
+}
+
+.hero {
+  margin-right: 40px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  gap: 12px;
+  margin-top: 86px;
+  margin-right: 40px;
+  width: 100%;
+  height: auto;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding-top: 20px;
+  margin-left: 20px;
+  left: 24px;
+  top: 10%;
+  padding-bottom: 40px;
+}
+
+.card img {
+  width: 100%;
+  height: 220px;
+  background: url(image_processing20200716-21735-f07s4k);
+  border-radius: 8px;
+}
+
+.card h2 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 34px;
+  line-height: 44px;
+  color: #172b4d;
+  text-align: left;
+}
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, I think it would be better to resize the title of your cards to replicate the Figma design by making it ```font-size: 32px``` as suggested 🙏